journal-of-paleontology.csl

```diff
@@ -205,11 +205,11 @@
             <text prefix=" " macro="publisher"/>
           </else-if>
           <else>
-            <text variable="title" suffix=". "/>
+            <text variable="title" suffix=": "/>
             <text variable="container-title" form="long" text-case="title" suffix=","/>
-            <group delimiter=":">
-              <text variable="volume" prefix=" "/>
-              <text variable="page"/>
+            <group delimiter=", ">
+              <text variable="volume" prefix="v. "/>
```

Comment:
same here -- better to put the v. and the p. into labels. For the p.: this is never going to be pp.?

Reply:
Apparently p. is to be used whether referring to one page or to a range.